troposphere/static/js/components/admin/ImageMaster.react.js

```diff
@@ -29,14 +59,21 @@ define(function (require) {
           return;
         }
 
+        var handleClick = function(){
+            this.onResourceClick(request);
+        }.bind(this);
```

Comment:
I believe that you could use a [fat arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) in the `onClick={handleClick}`. 

I'm okay with this.